### PR TITLE
Update Synology Cloud Station to v3482

### PR DESCRIPTION
**Pull request by martinbjeldbak**
- Set license
- Update download URL and sha256

### CODE REVIEW

File: Casks/synology-cloud-station.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 cask :v1 => 'synology-cloud-station' do
-  version '3475'
-  sha256 'ce777034689f05a2b03d937addeb2b2560f912288f8638df8188050672e55e7d'
+  version '3482'
+  sha256 '15bc32284c1466400291e863ba2541fd9b66c53d4ff2db9ca788f36a887314b4'
 
-  url "https://global.download.synology.com/download/Tools/CloudStation/#{version}/Mac/synology-cloud-station-#{version}.dmg"
+  url "http://global.download.synology.com/download/Tools/CloudStation/3.2-#{version}/Mac/Installer/synology-cloud-station-#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/synology-cloud-station.rb]: Adding numbers here completely defeats the purpose of interpolating `version`.

Comment (martinbjeldbak) [inline Casks/synology-cloud-station.rb]: So what do you suggest? Hardcoding '3482' into the url stanza and removing the 'version' stanza?

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 687fc04a8d35358ad9ab074cc29736684809b729. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

File: Casks/synology-cloud-station.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/synology-cloud-station.rb]: No, that wouldn’t make sense as well. [This](https://github.com/caskroom/homebrew-cask/commit/687fc04a8d35358ad9ab074cc29736684809b729) is the solution.

Comment (vitorgalvao) [inline Casks/synology-cloud-station.rb]: Also, the version stanza is required, it cannot be removed.

**Comment by martinbjeldbak** [comment]
Looks good... learning things every day. Thanks! :+1:

**Pull request closed without merging**